### DISCUSSION TITLE
Dead by Daylight

### DISCUSSION THREAD

**Discussion by beanoho** [cafali/SnapKey#91] [Compatibility List Submissions]
Please make it compatible for the multiplayer game Dead by Daylight!

**Comment by cafali** [comment]
Did you try running SnapKey as admin? DBD is on UE4, and SnapKey usually works great with games on that engine. I don’t have the game myself, so it’d be awesome if you could test it out.

What I’ve noticed so far is that some games with external launchers, like Valorant/GTA V, won’t work unless you start SnapKey as admin.

**Reply by beanoho** [reply]
I haven’t tested it out yet, haha. I think I would have to do it on an alt account with an extra copy of the game to be safe. I’m not sure if it will be detected by EAC even though I really would like to try it. :(

**Reply by cafali** [reply]
You won't get banned—there are no recorded bans for Snap Tap, SnapKey, or different software/hardware solutions. The only company that has released a statement about it is Valve, since this function has a bigger impact on CS2 - even in CS2, you won't be banned but only kicked from the server.

**Comment by beanoho** [comment]
Alright, thank you for the reassurance! I decided to test it out and it does work/is compatible on DbD. Thanks so much to the SnapKey dev team!

**Reply by cafali** [reply]
Thanks, DBD has been added to the [compatibility list](https://github.com/cafali/SnapKey/wiki/Compatibility-List)!

**Comment by kartoofy** [comment]
Super late, it works for dbd just fine.
